doc/classes/SpriteBase3D.xml

```diff
@@ -75,7 +75,8 @@
 			If [code]true[/code], texture is flipped vertically.
 		</member>
 		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate" default="Color(1, 1, 1, 1)">
-			A color value used to [i]multiply[/i] the texture's colors. Can be used for mood-coloring or to simulate the color of light.
+			A color value used to [i]multiply[/i] the texture's colors. Can be used for mood-coloring or to simulate the color of ambient light.
+			[b]Note:[/b] Unlike [member CanvasItem.modulate] for 2D, colors with values above [code]1.0[/code] (overbright) are not supported.
```

Comment:
[Calinou described a goord workaround](https://github.com/godotengine/godot/issues/75083#issuecomment-1475082169) but I feel like putting it here would bloat the description too much. Is it a common-enough to warrant giving example code?